lib/error_tracker.ex

```diff
@@ -107,6 +107,11 @@ defmodule ErrorTracker do
   * A `{kind, exception}` tuple in which case the information is converted to
   an Elixir exception (if possible) and stored.
   """
+
+  @type exception :: Exception.t() | {:error, any()} | {Exception.non_error_kind(), any()}
```

Comment:
Ideally this would live along the `context()` type defined in this same module and also have a `@typedoc` for consistency.
```suggestion
  @typedoc """
  An `Exception` or a `{kind, payload}` tuple compatible with `Exception.normalize/3`.
  """
  @type exception :: Exception.t() | {:error, any()} | {Exception.non_error_kind(), any()}
```